examples/resource_lacework_query/main.tf

```diff
@@ -0,0 +1,37 @@
+terraform {
+  required_providers {
+    lacework = {
+      source = "lacework/lacework"
+    }
+  }
+}
+
+  resource "lacework_query" "example" {
+    id    = var.query_id
+    query = <<EOT
+    {
```

Comment:
This type of query won't work until we release the feature request to have anonymous queries.